src/widgets/suwayomi/component.jsx

```diff
@@ -0,0 +1,41 @@
+import { useTranslation } from "next-i18next";
+
+import Container from "components/services/widget/container";
+import Block from "components/services/widget/block";
+import useWidgetAPI from "utils/proxy/use-widget-api";
+
+export default function Component({ service }) {
+  const { t } = useTranslation();
+
+  const { widget } = service;
+
+  const { data: suwayomiData, error: suwayomiError } = useWidgetAPI(widget);
+
+  if (suwayomiError) {
+    return <Container service={service} error={suwayomiError} />;
+  }
+
+  if (!suwayomiData) {
+    if (!widget.fields || widget.fields.length === 0) {
+      widget.fields = ["download", "nondownload", "read", "unread"];
+    } else if (widget.fields.length > 4) {
+      widget.fields = widget.fields.slice(0, 4);
+      widget.fields = widget.fields.map((field) => field.toLowerCase());
```

Comment:
```suggestion
      widget.fields = widget.fields.slice(0, 4);
```

Dont do the lowercase, it just allows user errors to go un-noticed.